Package: scssphp/scssphp v1.13.0
Base: 63d1157457e5554edf00b0c1fabab4c1511d2520
Source: OpenCart 8c9afc5a137aa22739c0a9479e70fa86f77d22ad
Purpose: PHP 8.4 explicit nullable parameter signatures

diff --git a/src/Compiler.php b/src/Compiler.php
index d654ee6..c7c1443 100644
--- a/src/Compiler.php
+++ b/src/Compiler.php
@@ -5054,3 +5054,3 @@ EOL;
      */
-    protected function multiplyMedia(Environment $env = null, $childQueries = null)
+    protected function multiplyMedia(?Environment $env = null, $childQueries = null)
     {
@@ -5146,3 +5146,3 @@ EOL;
      */
-    protected function pushEnv(Block $block = null)
+    protected function pushEnv(?Block $block = null)
     {
@@ -5210,3 +5210,3 @@ EOL;
      */
-    protected function set($name, $value, $shadow = false, Environment $env = null, $valueUnreduced = null)
+    protected function set($name, $value, $shadow = false, ?Environment $env = null, $valueUnreduced = null)
     {
@@ -5316,3 +5316,3 @@ EOL;
      */
-    public function get($name, $shouldThrow = true, Environment $env = null, $unreduced = false)
+    public function get($name, $shouldThrow = true, ?Environment $env = null, $unreduced = false)
     {
@@ -5381,3 +5381,3 @@ EOL;
      */
-    protected function has($name, Environment $env = null)
+    protected function has($name, ?Environment $env = null)
     {
diff --git a/src/Formatter.php b/src/Formatter.php
index 6137dc6..c477e6f 100644
--- a/src/Formatter.php
+++ b/src/Formatter.php
@@ -274,3 +274,3 @@ abstract class Formatter
      */
-    public function format(OutputBlock $block, SourceMapGenerator $sourceMapGenerator = null)
+    public function format(OutputBlock $block, ?SourceMapGenerator $sourceMapGenerator = null)
     {
diff --git a/src/Parser.php b/src/Parser.php
index e7cb2e8..674a080 100644
--- a/src/Parser.php
+++ b/src/Parser.php
@@ -142,3 +142,3 @@ class Parser
      */
-    public function __construct($sourceName, $sourceIndex = 0, $encoding = 'utf-8', Cache $cache = null, $cssOnly = false, LoggerInterface $logger = null)
+    public function __construct($sourceName, $sourceIndex = 0, $encoding = 'utf-8', ?Cache $cache = null, $cssOnly = false, ?LoggerInterface $logger = null)
     {
diff --git a/src/Warn.php b/src/Warn.php
index 592b44c..7ad0615 100644
--- a/src/Warn.php
+++ b/src/Warn.php
@@ -61,3 +61,3 @@ final class Warn
      */
-    public static function setCallback(callable $callback = null)
+    public static function setCallback(?callable $callback = null)
     {
diff --git a/src/Node/Number.php b/src/Node/Number.php
index 6c04458..a38ba5f 100644
--- a/src/Node/Number.php
+++ b/src/Node/Number.php
@@ -580,3 +580,3 @@ class Number extends Node implements \ArrayAccess, \JsonSerializable
      */
-    public function output(Compiler $compiler = null)
+    public function output(?Compiler $compiler = null)
     {
